### PR TITLE
More updates to cookiecutter

### LINKED ISSUE #67: Upgrade Celery to 4.0

### LINKED ISSUE #68: Updates to webpack shell scripts
Look here: https://github.com/mitodl/micromasters/pull/3278

### PR DESCRIPTION
**Pull request by noisecapella**
Fixes #67 
Fixes #68 

Some more changes backported from Micromasters:
 - Celery upgraded to 4.x
 - README changes
 - Updated `webpack_dev_server.sh` and added `scripts/envs.sh`
 - Enable Django debug toolbar

### CODE REVIEW

File: {{ cookiecutter.project_name }}/webpack_dev_server.sh [hunk 1]
```diff
@@ -1,36 +1,21 @@
...
+# Define some environment variables to figure out where we are running
+source ./scripts/envs.sh
 
-if [ -z "$WEBPACK_DEV_SERVER_PORT" ] ; then
-  WEBPACK_PORT='{{ cookiecutter.webpack_dev_port }}'
-else
-  WEBPACK_PORT="$WEBPACK_DEV_SERVER_PORT"
-fi
+WEBPACK_HOST='0.0.0.0'
+WEBPACK_PORT='8078'
```
Comment (mbertrand) [inline {{ cookiecutter.project_name }}/webpack_dev_server.sh]: In cookiecutter.json, webpack_dev_port is set to 8072 and when I load the index page, 'style.js' and 'root.js' javascript files are successfully loaded from http://0.0.0.0:8072 but '__webpack_hmr' tries to load from http://0.0.0.0:8078 and fails (refused to connect).  If I change the above to 8072 then it loads successfully.

Comment (noisecapella) [inline {{ cookiecutter.project_name }}/webpack_dev_server.sh]: Ah good catch

**Comment by noisecapella** [comment]
Should be ready to review again

**Comment by mbertrand** [comment]
👍 

Latest travis tests failed but it looks like a travis issue not a PR issue.

**Pull request merged**